src/App.tsx

```diff
@@ -47,86 +26,12 @@ function App() {
     setIsLogedin(accessToken ? true : false);
   }, [accessToken]);
 
-  useEffect(() => {
-    loadData();
-  }, []);
-
-  const loadData = async () => {
-    setLoading(true);
-    await axiosInstance
-      .get("/todos")
-      .then((res) => dispatch(setTodoAction(res.data)));
-    await axiosInstance
-      .get("/category")
-      .then((res) => dispatch(setCategoryAction(res.data)));
-    await axiosInstance
-      .get("/status")
-      .then((res) => dispatch(setStatusAction(res.data)));
-    setLoading(false);
-  };
-
-  const onLogoutSuccess = () => {
-    setIsLogedin(false);
-    window.localStorage.removeItem("accessToken");
-  };
-
   return (
     <>
-      <div className="welcome-text">
-        {state.isLogedin && (
-          <span>
-            Welcome - {userName} |{" "}
-            <Button label="Logout" onClick={onLogoutSuccess} varient="link" />
-          </span>
-        )}
-      </div>
-      {tost.tostState ? (
-        <Tost
-          delay={3000}
-          setTost={setTost}
-          tostType={tost.tostType}
-          tostMessage={tost.tostMessage}
-        />
-      ) : (
-        ""
-      )}
-      <div className="container">
-        <div className="todo-container">
-          {loading ? <Loader /> : ""}
-          <h3 className="text-center">{formTitle}</h3>
-          {state.isLogedin && (
-            <div>
-              <TodoForm
-                todos={state.tasks}
-                status={state.status}
-                setActiveId={setActiveId}
-                activeId={activeId}
-                setFilter={setFilter}
-                setLoading={setLoading}
-              />
+      <Header setIsLogedin={setIsLogedin} isLogedin={state.isLogedin} />
```

Comment:
Do we need to send state to header, as we are using redux for state management

Reply:
Updated